HW4/task1.py

```diff
@@ -0,0 +1,17 @@
+# 1. Створити програму, яка буде очікувати введення тексту від користувача і повідомляти — чи є
+# введений текст “числом” чи “словом”.
+# 2. Якщо введений текст “число”, програма має також вказати чи є воно парним чи непарним.
+# 3. Якщо це “слово”, програма має вказати його довжину.
+
+user_text = input('Please, type something: ')
+if user_text.isdigit() == True:
```

Comment:
the trick was to use `type` function
```suggestion
if type(user_text) == int:
```